### DISCUSSION TITLE
Permission Settings - Restrictions for "Other Authors' ..."

### DISCUSSION THREAD

**Discussion by brankojay** [statamic/cms#4593] [Q&A] [answered]
Hi all!

I would like to create different user groups (e.g. Team A, Team B) based on the same set of permissions (e.g. Editor Team). Users within those groups should not be able to edit or delete entries of a specific collection created by users of other groups.

I expected to achieve this with the following settings:

<img width="498" alt="Bildschirmfoto 2021-10-27 um 18 40 26" src="https://user-images.githubusercontent.com/14178393/139116719-bed134c9-6e99-4183-b591-7bd43cefa936.png">

This setup is not working (tested with v3.2.18). I can still edit or delete entries created by users from other groups. Am i missing something or maybe misinterpreting "other authors'"?

Many thanks in advance!

**Comment by jesseleite** [comment]
Are you logged in as a super user by chance?

**Comment by brankojay** [comment]
@jesseleite thank you for the reply. To test my setup, I have created two new users without super admin capabilities. Each of them is in another user group (User A -> Team A, User B -> Team B). As an additional info: they are also capable of editing and deleting entries created by my super admin…

**Reply by jesseleite** [reply]
The entries in question have an `author` field, with an author properly assigned in the data?  ie)

![CleanShot 2021-10-27 at 15 49 47](https://user-images.githubusercontent.com/5187394/139136822-a448b74f-6b34-4ac1-b003-f823a2cda474.png)

In the above example, Bob will not be able to edit entry 1 there, because it's authored by Jesse. Bob shouldn't be able to edit entry 2 or 3 there either, but if he can edit any of them, the entry blueprint likely doesn't have a users field with the handle of `author` (singular).

![CleanShot 2021-10-27 at 15 54 00](https://user-images.githubusercontent.com/5187394/139137378-3998e2c4-5eef-44d1-9479-a5f0c9c827a0.png)

You can read more about this here: https://statamic.dev/upgrade-guide/3-0-to-3-1#entry-author-permissions

**Comment by brankojay** [comment, marked as answer]
@jesseleite you got it! Thank you so much for your time and the kind explanation. The mentioned `author` field was not present in the blueprint. After adding it, the logic applied to all new entries.

As a side note: It‘s not quite what I was trying to achieve, because the permissions are applied on user-level and not on user-group-level. But now that’s exactly what it says in the permission settings. The possibility to apply that behavior on user-group-level is maybe something which will come in a future release?

**Reply by jesseleite** [reply]
You got it, the author field is author-specific permissions on a user-level. We do plan on fleshing out permissions more. I believe [per-site permissions](https://github.com/statamic/cms/issues/2667) are next on the list. Not sure there's an issue for per-entry-by-group permissions yet though; Feel free to [create one here](https://github.com/statamic/ideas/issues/new).

PS. If you want to get crafty, it _**MAY**_ be possible with a custom fieldtype that stores a user group handle to the `author` field on an entry, but augments to a list of users when accessed by the policy. I'm not 100% sure if this would work, but might be worth a try?

**Reply by brankojay** [reply]
Ok, I will look out if my requirements might be useful enough for the community to propose them as a feature request.

Thank you for the hint to the crafty adventure…I can see in the extended docs that it‘s possible to define a custom policy where i can access data from an entry. Right now I‘m approx. 24h into statamic (and already in love), so I will travel a bit more along the path of the basics and come back to the hacky stuff later… :)
